production/helm/loki/templates/admin-api/deployment-admin-api.yaml

```
@@ -58,6 +58,9 @@ spec:
       hostAliases:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if semverCompare ">=1.33-0" .Capabilities.KubeVersion.Version }}
+      hostUsers: {{ .Values.adminApi.hostUsers }}
+      {{- end }}
```

The `(semverCompare ">=1.33-0" (include "loki.kubeVersion" .))` pattern is used in other parts of the chart.

WDYT about this:

```suggestion
      {{- if and (semverCompare ">=1.33-0" (include "loki.kubeVersion" .)) (ne .Values.adminApi.hostUsers nil) }}
      hostUsers: {{ .Values.adminApi.hostUsers }}
      {{- end }}
```

I would like to get end users the chance to not define `hostUsers`. Sometimes, restrictive polices are present and end users should able to unset the `hostUsers` property.

---

done